src/lib/journey/callbacks/suspend-text-output/suspend-text-output.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  import type { SuspendedTextOutputCallback } from '@forgerock/javascript-sdk';
+  import Centered from '$components/primitives/box/centered.svelte';
```

Comment:
I try to separate the external dependency imports from the internal components with an empty line.